### PR TITLE
add data science client certs

### LINKED ISSUE #40: Certificate creation and cert/key storage
With the use of TLS we need a signed server certificate and client CA root cert with which to issue a signed client cert to the Data Science team. The keys and certs need to be stored securely. 

We're leaving the larger process of issuing new certs and onboarding Data Providers to future work.

### PR DESCRIPTION
**Pull request by cbrit**
Closes #40

### CODE REVIEW

File: certs/client.crt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+-----BEGIN CERTIFICATE-----
```
Comment (EricBolten) [inline certs/client.crt]: We only need the CA cert to validate the connection. The data provider can update their cert or issue many for different machines as long as they are from the same CA.

**Review by EricBolten — COMMENTED** [review]
Comment re: certs. It would also be wise to explain in the README or elsewhere why the config is set up the way it is to use this checked in client root CA.

File: steward/src/commands/cosmos_mode.rs [hunk 2]
```diff
@@ -33,15 +33,19 @@ impl Runnable for CosmosSignerCmd {
...
                     std::process::exit(1)
                 });
 
             // run it
-            let addr = ([127, 0, 0, 1], STEWARD_PORT).into();
+            let port = match config.server.port {
+                Some(port) => port,
+                None => DEFAULT_STEWARD_PORT,
+            };
+            let addr = ([127, 0, 0, 1], port).into();
```
Comment (EricBolten) [inline steward/src/commands/cosmos_mode.rs]: To answer a question I believe you asked out of band, yes, we should also make the listen address configurable.

**Review by EricBolten — COMMENTED** [review]
One last comment re: config.

File: steward/src/commands/cosmos_mode.rs (hunk 2 above)
Comment (cbrit) [reply steward/src/commands/cosmos_mode.rs]: ~~Including domain name or just IP?~~

nevermind. Addressed this

**Review by EricBolten — APPROVED** [review]
LGTM

**Pull request merged**